.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: php
 
+env:
+    global:
+        - SYMFONY_DEPRECATIONS_HELPER: weak
```

Comment:
To hide the deprecations of symfony < 2.8, [this](https://github.com/symfony/swiftmailer-bundle/pull/119/files#diff-354f30a63fb0907d4ad57269548329e3R29) reactivate the deprecations for symfony 3.0.